app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java

```diff
@@ -215,6 +217,19 @@ private void importDatabase(final String filePath) {
                 throw new Exception("Could not create databases dir");
             }
 
+            if (!manager.isValidBackupFile(filePath)) {
+                final Snackbar snackbar = Snackbar.make(
+                        requireActivity().findViewById(android.R.id.content),
+                        getString(R.string.no_valid_zip_file), Snackbar.LENGTH_LONG);
+                snackbar.setDuration(3000);
+                snackbar.setAction(R.string.show_info, v -> {
+                    snackbar.dismiss();
+                    onError(new Exception("The provided zip was not created by NewPipe."));
```

Comment:
The additional info is that the zip file is not created by NewPipe as you can see. The SnackBar text itself shows that the zip is not supported, the exception gives some more information (reason).

Reply:
Can't that be also squeazed into the snackbar, since only a single line of text?

Besides that: Since the message is intented for users and not developers, it needs to be translatable